containers/php/Dockerfile

```diff
@@ -13,12 +13,23 @@ ENV PHP_VERSION=${PHP_VERSION} \
     PHP_MAX_INPUT_VARS=1500 \
     PHP_ASSERT=-1 \
     PHP_XDEBUG_HOST=host.docker.internal \
-    PHP_XDEBUG_MODE=off
+    PHP_XDEBUG_MODE=off \
+    TIDEWAYS_APIKEY="" \
+    TIDEWAYS_DAEMON="tcp://tideways-daemon:9135" \
+    TIDEWAYS_SAMPLERATE=25 \
+    TIDEWAYS_SERVICE="web"
```

Comment:
Does tideways still use "cli" for command line runs? Or will this env var override that?